docs/dwarf-op.rst

```diff
@@ -1,158 +1,180 @@
+``unprotected``
+    Selects unprotected dwarves.
+``drunks``
+    Selects any dwarves who have the ``DRUNK`` profession, including those who
+    have been zeroed by the ``--clear`` command option.
+``jobs "[" jobs <jobname> [<jobname> ...] "]"``
+    Selects any dwarves with the specified custom professions.
+``waves "[" waves <num> [<num> ...] "]"``
+    Selects dwarves from the specified migration waves. Waves are enumerated
+    starting at 0 and increasing by 1 with each wave. The waves go by season and
+    year and thus should match what you see in `list-waves` or Dwarf Therapist.
+    It is recommended that you ``--show`` the selected dwarves before modifying
+    them.
+
+Options
+-------
+
+``--reset``
+    Forget which dwarves have been optimized. However, if you reload an existing
+    save, the optimization list will be reloaded.
+``--resetall``
+    Forget which dwarves have been optimized and remove the persisted
+    optimization data.
+``--show``
+    Lists the selected dwarves. Useful for previewing selected dwarves before
+    modifying them or looking up the migration wave number for a group.
+``--clean <value>``
+    Checks for skills with a rating of ``<value>`` and removes them from the
+    dwarf's skill list.
+``--clear``
+    Zeroes the skills and attributes of selected dwarves. No attributes, no
+    labors. Assigns ``DRUNK`` profession.
+``--reroll [inclusive]``
+    Clears attributes of selected dwarves, then rerolls that dwarf based on
+    their jobs. Run ``dwarf-op --list attrib_levels`` to see how stats are
+    distributed. If ``inclusive`` is specified, then attributes are not cleared,
+    but rather will only be changed if the current reroll is better. This
+    command ignores dwarves with jobs that are not listed in the ``jobs`` table.
+``--optimize``
+    Performs a job search for unoptimized dwarves. Run
+    ``dwarf-op --list job_distribution`` to see how jobs are distributed.
+``--applyjobs "[" <job> [<job> ...] "]"``
+    Applies the listed jobs to the selected dwarves. Run
+    ``dwarf-op --list jobs`` to see available jobs.
+``--applyprofessions "[" <profession> [<profession> ...] "]"``
+    Applies the listed professions to the selected dwarves. Run
+    ``dwarf-op --list professions`` to see available professions.
+``--applytypes "[" <profession> [<profession> ...] "]"``
+    Applies the listed types to the selected dwarves. Run
+    ``dwarf-op --list dwf_types`` to see available types.
+``--renamejob <name>``
+    Renames the selected dwarves' custom professions to the specified name.
+
+.. _dorf_tables:
+
+Data tables
+-----------
+
+The data tables that ``dwarf-op`` uses are described below. They can be
+inspected with ``dwarf-op --list <table name>``.
+
+``job_distributions``
+    Defines thresholds for each column of distributions. The columns must add up
+    to the values in the thresholds row for that column.  Every other row
+    references an entry in the ``jobs`` table.
```

Comment:
This is incorrect. The values in the columns **should** add up to (not technically required) the values in the thresholds row. Not very important to make that distinction though.

Reply:
what happens if they don't add up right?

Reply:
I'll double check, but less than will possibly crash when it tries to pull the first larger than. If it's larger than though, it should keep chugging just fine since it can keep pulling jobs.